test/jdk/com/sun/jdi/EATests.java

```diff
@@ -271,7 +271,7 @@ public static class TargetVMOptions {
         public final boolean EliminateAllocations;
         public final boolean DeoptimizeObjectsALot;
         public final boolean DoEscapeAnalysis;
-        public final boolean ZGCIsSelected;
+        public final boolean ConcurrentGCIsSelected;
```

Comment:
Let's do explicit `ZGCIsSelected` and `ShenandoahIsSelected`. Unfortunately, "Concurrent GC" is fairly ambiguous.